config/kubernetes/org.yaml

```diff
@@ -1523,6 +1523,8 @@ teams:
     - MrHohn
     - rramkumar1
     - thockin
+    - prameshj
+    - swetharepakula
```

Comment:
Can you please ensure that the members list is sorted after the addition?

Reply:
ack